### PR TITLE
Fix Hussein's bug

### PR DESCRIPTION
**Pull request by scottfrazer**

### CODE REVIEW

File: src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala [hunk 1]
```diff
@@ -146,14 +146,14 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
...
-    def logMapFromStatusMap(statusMap: Map[FullyQualifiedName, ExecutionStatus]): Try[Map[LocallyQualifiedName, StdoutStderr]] = {
+    def logMapFromStatusMap(statusMap: Map[FullyQualifiedName, ExecutionStatus]): Try[Map[FullyQualifiedName, StdoutStderr]] = {
       Try {
         val callsToPaths = for {
           (call, status) <- statusMap.toSeq
           if Set(ExecutionStatus.Done, ExecutionStatus.Failed).contains(status)
           (wf, callLqn) = assertCallFqnWellFormed(call).get
           callStandardOutput = backend.stdoutStderr(workflowId, wf, callLqn)
-        } yield callLqn -> callStandardOutput
+        } yield call -> callStandardOutput
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: This relies on `Call#toString` to return the FQN?

Comment (scottfrazer) [inline src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: call is actually a `FullyQualifiedName` in this context

**Comment by mcovarr** [comment]
:+1:, over to @horneth

**Comment by Horneth** [comment]
Looks good to me :) I wasn't sure if it's still necessary to split the `call` FQN in `(wf, callLqn)` but it looks like it's useful to have it this way to retrieve the `callStandardOutput`.

**Pull request merged**